src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterHelper.cs

```diff
@@ -199,35 +202,37 @@ public static void UpdateCoreTypeBlobs(string testDataFilePath, string[] blobs)
             File.WriteAllLines(testDataFilePath, updatedTestDataLines);
         }
 
-        public static byte[] SerializeObjectToRaw(object obj)
+        public static byte[] SerializeObjectToRaw(object obj, FormatterAssemblyStyle assemblyStyle = FormatterAssemblyStyle.Simple)
```

Comment:
I think no default is better here since we are testing this we always want to check all values